Tella/Data/Database/TellaDataBase.swift

```diff
@@ -44,6 +44,9 @@ class TellaDataBase : DataBase {
                 createGDriveServerTable()
                 createGDriveReportTable()
                 createGDriveReportFilesTable()
+                createNextcloudServerTable()
```

Comment:
let's move nextcloud methods to a new case statement with version 6.

Don't forget to:
[x] add fallthrough to case 5
[x]  update `databaseVersion` to 7 in `DatabaseConstants`

Reply:
Changing the databaseVersion to 6 should be done when we provide the production version